### ISSUE TITLE
Question about the URL request

### ISSUE THREAD

**Issue by dannypage** [sportsdataverse/fastRhockey#39]
Curious why this line is needed in the code - or more broadly, why there's a Callback at the end of the base_url? It seems to work without the callback, but maybe there's some information I am missing.

I'm also curious about the key and if we should be concerned about it ever rotating. Thanks!

https://github.com/sportsdataverse/fastRhockey/blob/12ff24b40a665c6b47bcfb757e207ecec22d3eff/R/pwhl_teams.R#L28

**Comment by benhowell71** [comment]
Yeah, as I'm going back and updating / re-writing some of the functions, I am converting them to this to handle the regex better: 
```
  callback_pattern <- "angular.callbacks._\\d+\\("
  res <- gsub(callback_pattern, "", res)
```

**Comment by saiemgilani** [comment]
Answering and closing: the `callback=` parameter is HockeyTech's JSONP envelope — the feed wraps its JSON in `angular.callbacks._N(...)`, so the package requests a fixed callback and strips the wrapper. Since v1.0.0 this is centralized in a single helper (`R/hockeytech_helpers.R`) that strips any `angular.callbacks._N` prefix rather than per-function regex. On key rotation: these are the public per-league keys shipped in each league site's own JavaScript, and they historically rotate by *addition* — old key generations keep working — so breakage risk is low; all keys now live in one registry so a rotation would be a one-line fix. Thanks for the question!
